tests/README.md

```diff
@@ -153,30 +163,81 @@ In order to run the tests in GCP, you need a few things:
   - this assumes that your GKE cluster is using private nodes. If using public nodes, you don't need this.
 - GCP VM and firewall rule to send ingress traffic to GKE
 
+To just set up the VM with no router (this will not run the tests):
+
+```makefile
+make create-and-setup-vm
+```
+
+Otherwise, you can set up the VM, router, and run the tests with a single command. See the options in the sections below.
+
+By default, the tests run using the version of NGF that was `git cloned` during the setup. If you want to make
+incremental changes and copy your local changes to the VM to test, you can run
+
+```makefile
+make sync-files-to-vm
+```
+
+#### Functional Tests
+
 To set up the GCP environment with the router and VM and then run the tests, run the following command:
 
 ```makefile
 make setup-gcp-and-run-tests
 ```
 
-If you just need a VM and no router (this will not run the tests):
+To use an existing VM to run the tests, run the following
 
 ```makefile
-make create-and-setup-vm
+make run-tests-on-vm
+```
+
+#### NFR tests
+
+To set up the GCP environment with the router and VM and then run the tests, run the following command:
+
+
+```makefile
+make setup-gcp-and-run-nfr-tests
 ```
 
 To use an existing VM to run the tests, run the following
 
 ```makefile
-make run-tests-on-vm
+make nfr-test
 ```
 
+##### Longevity testing
+
+This test is run on its own (and also not in a pipeline) due to its long-running nature. It will run for 4 days before
+the tester must collect the results and complete the test.
+
+To start the longevity test, set up your VM (`create-and-setup-vm`) and run
+
+```makefile
+make start-longevity-test
```

Comment:
would it make sense to make configurable the period of longevity test? so we can shorten for example, or extend it?

Reply:
I can, just means another ENV var to pass through all the scripts.

Other option is a user updates `longevity-wrk.sh` and syncs their files, which is still easy to do. I don't suspect we'll be changing this time frequently enough to warrant a dedicated variable?

Reply:
would it make sense to add a simple note then. Something like, to change the test period, update longevity-wrk.sh file.